core/modules/filters/savers.js

```diff
@@ -0,0 +1,24 @@
+/*\
+title: $:/core/modules/filters/savers.js
+type: application/javascript
+module-type: filteroperator
+
+This Filter operator return savers title
+
+\*/
+;(function () {
+  /*jslint node: true, browser: true */
+  /*global $tw: false */
+  'use strict'
+
+  /*
+  Export our filter function
+  */
+  exports.savers = function (source, operator, options) {
```

Comment:
Note that coding style is to omit whitespace between `function` and the opening parenthesis, and to omit whitespace after commas in argument lists.

Reply:
General Comment below